internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,71 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
 )
 
 const (
 	// Длина одного шага в метрах
 	stepLength = 0.65
 	// Количество метров в одном километре
 	mInKm = 1000
+	// Ошибка
+)
+
+var (
+	ErrorWrongFormat = errors.New("Неверный формат данных")
+	ErrorWrongSteps  = errors.New("Неверное количество шагов")
+	ErrorWrongTime   = errors.New("Неверный формат времени")
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// преобразование входящей строки в слайс
+	dataSlice := strings.Split(data, ",")
+	//
+	if len(dataSlice) != 2 {
+		return 0, 0, ErrorWrongFormat
+	}
+	steps, err := strconv.Atoi(dataSlice[0])
+	// Проверка "была ли ошибка"
+	if err != nil {
+		return 0, 0, err
+	}
+	if steps <= 0 {
+		return 0, 0, ErrorWrongSteps
```

Comment:
Лучше четко писать в чем проблема
```suggestion
		return 0, 0, fmt.Errorf("steps must be positive")
```